libs/aws/Makefile

```
@@ -26,34 +48,42 @@ lint_package: PYTHON_FILES=langchain_aws
 lint_tests: PYTHON_FILES=tests
 lint_tests: MYPY_CACHE=.mypy_cache_test
 
-lint lint_diff lint_package lint_tests:
-	poetry run ruff .
+lint:         ## Run linter
+	poetry run ruff check
+
+lint_diff:    ## Run linter
 	poetry run ruff format $(PYTHON_FILES) --diff
-	poetry run ruff --select I $(PYTHON_FILES)
+
+lint_package: ## Run linter on Package
+	poetry run ruff check --select I $(PYTHON_FILES)
+
+lint_tests:   ## Run linter tests
 	mkdir -p $(MYPY_CACHE); poetry run mypy $(PYTHON_FILES) --cache-dir $(MYPY_CACHE)
 
-format format_diff:
+format format_diff: ## Run code formatter
 	poetry run ruff format $(PYTHON_FILES)
-	poetry run ruff --select I --fix $(PYTHON_FILES)
+	poetry run ruff check --select I --fix $(PYTHON_FILES)
 
-spell_check:
+spell_check: ## Run code spell check
 	poetry run codespell --toml pyproject.toml
 
-spell_fix:
+spell_fix: ## Run code spell fix
 	poetry run codespell --toml pyproject.toml -w
 
-check_imports: $(shell find langchain_aws -name '*.py')
-	poetry run python ./scripts/check_imports.py $^
+######################
+# DEPENDENCIES
+######################
+
+install_deps: ## Install tool dependencies
+	@pip install --no-cache -U poetry
+	@poetry install --with test,codespell,lint
```

Missing some targets.
```suggestion
	@poetry install --with dev,test,codespell,lint,typing
```